tcms/telemetry/templates/telemetry/testing/execution-dashboard.html

```diff
@@ -27,6 +27,8 @@
                     <th>{% trans "Summary" %}</th>
                     <th>{% trans "Status" %}</th>
                     <th>{% trans "Build" %}</th>
+                    <th>{% trans "Product" %}</th>
```

Comment:
The natural order here is `Product` and then `Build`. Same applies for the ordering in `drawTable()`.